src/components/CardLists/progress.js

```diff
@@ -0,0 +1,21 @@
+import useStore from '../../hooks/useStore';
+import ProgressHistoryCard from '../Cards/progressHistoryCard';
+
+import StyledCardlist from './styled';
+
+export default function CardlistProgress() {
+	const exerciseCards = useStore(state => state.exerciseCards);
+
+	return (
+		<>
+			<h1>Progress History</h1>
+			<StyledCardlist>
+				{exerciseCards.map(exerciseCard => {
+					return (
+						<ProgressHistoryCard key={exerciseCard.id} exerciseCard={exerciseCard} />
```

Comment:
It's not really clear if you are returning a list or something else. Looking inside `<ProgressHistoryCard>`you are returning a list width the name and other things, but not a list containing all the elements. I'm not sure how it would look semantically. Please take a look at it.